packages/tasks/src/tasks/depth-estimation/data.ts

```diff
@@ -24,8 +28,8 @@ const taskData: TaskDataCustom = {
 	metrics: [],
 	models: [
 		{
-			description: "Strong Depth Estimation model trained on 1.4 million images.",
-			id: "Intel/dpt-large",
+			description: "Cutting-edge depth estimation model.",
```

Comment:
QQ: Does adding the Depth Anything v2 optimised for Coreml here as a point of reference and as a kickback to apple org makes sense?

Could also be DETR too

cc: @osanseviero too

Reply:
feel free to make a commit suggestion on a separate row, and I find it relevant so would be nice addition